lib/dictBuilder/zdict.h

```diff
@@ -86,6 +86,7 @@ typedef struct {
     unsigned d;                  /* dmer size : constraint: 0 < d <= k : Reasonable range [6, 16] */
     unsigned steps;              /* Number of steps : Only used for optimization : 0 means default (32) : Higher means more parameters checked */
     unsigned nbThreads;          /* Number of threads : constraint: 0 < nbThreads : 1 means single-threaded : Only used for optimization : Ignored if ZSTD_MULTITHREAD is not defined */
+    double splitPoint;           /* Percentage of samples used for training: the first nbSamples * splitPoint samples will be used to training */
```

Comment:
What is the default value?

Reply:
Right now the splitPoint is default to 0.8. I store it in a constant named DEFAULT_SPLITPOINT. I use this constant when I initialize the parameters because I have not yet decided on the "best" splitPoint and just save the default value as a constant so it's easier to change it later. Would you recommend having the actual value of 0.8 instead of the stored constant? It seems like the actual values are used when d and steps are initialized in ZDICT_cover_params_t defaultCoverParams(void)in zstdcli.c.

Reply:
You should mention the default value "0.0 means default (0.8)" or whatever we decide the default is.